src/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import './Calculator.css';
+
+function Calculator() {
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class based components (All components should be class-based).`", kindly change your `Calculator component`, which is currently a `functional component` to a `class-based` component.